article-store/events/utils.py

```diff
@@ -80,7 +80,7 @@ def wrapper(*args, **kwargs):
 
 
 @setup_exchanges
-def send_article_message(msg_type: str, run_id: str, message: Optional[str]) -> None:
+def send_article_message(msg_type: str, run_id: str, message: Optional[str] = '') -> None:
```

Comment:
`None` rather than an empty string?

Reply:
Could argue it both ways, it's not being manipulated in the function so it `None` would still work.